### PR TITLE
replaced individual lodash packages with original lodash package

### PR DESCRIPTION
**Pull request by seeden**
TODO:

- [x] If this PR is a new feature, reference an issue where a consensus about the design was reached (not necessary for small changes)
- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass
- [x] Update CHANGELOG.md with your change
- [x] Add your name and email to the AUTHORS file (optional)
- [x] If this was a change that affects the external API, update the docs and post a link to the PR in the discussion

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -48,3 +48,4 @@ Agustin Polo <poloagustin@gmail.com>
 Chris Metz <metzc@users.noreply.github.com>
 jon wong <jon@coursera.org>
 jon wong <j@jnwng.com>
+Zlatko Fedor <zlatkofedor@cherryprojects.com>
```
Comment (gajus) [inline AUTHORS]: Unrelated to this PR, but whats the purpose of this file?

There is already a contributors tab that lists all the people who have contributed code.

Comment (seeden) [inline AUTHORS]: @gajus pure happiness you need to try it

Comment (helfer) [inline AUTHORS]: @gajus it's basically for this: https://github.com/apollostack/apollo-client/commit/7f30fe6a845d31cf76eb8634c2bdfad3b00396a7

**Comment by helfer** [comment]
Awesome, thanks a lot @seeden !

**Pull request merged**